deploy/examples/ApplicationMonitoring.yaml

```diff
@@ -11,3 +11,5 @@ spec:
   prometheusInstanceNamespaces: application-monitoring
   alertmanagerInstanceNamespaces: application-monitoring
   selfSignedCerts: true
+  priorityClassName: "managed-service-priority"
```

Comment:
Is this correct, or should it be `rhoam-pod-priority` maybe?

Reply:
yes, I thought I had updated that.